hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/decommissionSummary.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Descriptions, Popover, Tooltip } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { withRouter } from 'react-router-dom';
+import { RouteComponentProps } from 'react-router';
+import axios from 'axios';
+import { showDataFetchError } from '@/utils/common';
+
+interface IDecommissionSummaryProps extends RouteComponentProps<object> {
+  uuid: string;
+  isLoading: boolean;
+  summaryData: string[];
+  record: string[];
+}
+
+class DecommissionSummary extends React.Component<IDecommissionSummaryProps> {
+  constructor(props = {}) {
+    super(props);
+    this.state = {
+      uuid: this.props.uuid,
+      record: this.props.record,
+      isLoading: false,
+      summaryData: []
+    };
+  }
+
+  componentDidMount(): void {
+    this.setState({
+      isLoading: true
+    });
+    if (this.state.record && this.state.summaryData !== 'null' && this.state.summaryData !== 'undefined') {
```

Comment:
I think this can be simplified to:
```suggestion
    if (this.state?.record && this.state?.summaryData) {
```

Reply:
Done the changes in latest commit3e08fb7b48adea2a78e0f77408427da00db453bf and done testing with local cluster data.